### DISCUSSION TITLE
Application cannot find resources js/css (looking for _next folder instead of .next)

### DISCUSSION THREAD

**Discussion by abhinavgurung** [vercel/next.js#62828] [Help]
### Summary

I have deployed a nextjs application on k8s cluster. The page is loading, but the resources are not found. It is looking for files under `_next` folder. But when I sh in to pod, I can see `.next` folder.
![img1](https://github.com/vercel/next.js/assets/12020147/d64abed1-f98f-49ec-8c7b-134a7eb21f67)

### Additional information

```js
the docker file used:


FROM node:18-alpine AS base

# Install dependencies only when needed
FROM base AS deps
# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
RUN apk add --no-cache libc6-compat
WORKDIR /app

# Install dependencies based on the preferred package manager
COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
RUN \
  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
  elif [ -f package-lock.json ]; then npm ci; \
  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i --frozen-lockfile; \
  else echo "Lockfile not found." && exit 1; \
  fi


# Rebuild the source code only when needed
FROM base AS builder
WORKDIR /app
COPY --from=deps /app/node_modules ./node_modules
COPY . .

# Next.js collects completely anonymous telemetry data about general usage.
# Learn more here: https://nextjs.org/telemetry
# Uncomment the following line in case you want to disable telemetry during the build.
ENV NEXT_TELEMETRY_DISABLED 1

RUN npm run build

# Production image, copy all the files and run next
FROM base AS runner
WORKDIR /app

ENV NODE_ENV production
# Uncomment the following line in case you want to disable telemetry during runtime.
# ENV NEXT_TELEMETRY_DISABLED 1

RUN addgroup --system --gid 1001 nodejs
RUN adduser --system --uid 1001 nextjs

COPY --from=builder /app/public ./public

# Set the correct permission for prerender cache
RUN mkdir .next
RUN chown nextjs:nodejs .next

# Automatically leverage output traces to reduce image size
# https://nextjs.org/docs/advanced-features/output-file-tracing
COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static

USER nextjs

EXPOSE 3000

ENV PORT 3000
# set hostname to localhost
ENV HOSTNAME "0.0.0.0"

# server.js is created by next build from the standalone output
# https://nextjs.org/docs/pages/api-reference/next-config-js/output
CMD ["node", "server.js"]
```
```


### Example

_No response_

**Comment by icyJoseph** [comment]
Hi,

That's juts a prefix for the so called, next internals requests.

Are you perhaps using an assetPrefix, or basePath?

**Reply by abhinavgurung** [reply]
No in my Application, I am not using any of those. I only have output:standalone in my next.config file.

**Reply by icyJoseph** [reply]
Alright, then assuming there's nothing funny going on with the paths, we should probably look at the Docker. Have you used the Next.js recommended Docker file for standalone mode?

**Reply by abhinavgurung** [reply]
Hi @icyJoseph  
I have posted my docker file above. 
i got this from

https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile

**Reply by icyJoseph** [reply]
I am still thinking that this might be a problem with the application path.

Is the app running at the root of the domain?

Does this work if you run the production mode, locally?

**Reply by abhinavgurung** [reply]
Yes the app runs locally very well. 
my domain is something like this
https://mydomain.abc.de.ca

and as shown in the image above, 
It is trying to access js resources at 
https://mydomain.abc.de.ca/_next/static/chunks…..

**Reply by abhinavgurung** [reply]
Also yes the app is running at the root of the domain.

**Reply by abhinavgurung** [reply]
@icyJoseph any suggestions?

**Reply by icyJoseph** [reply]
Not really, I'd guess this has to be a CDN issue.

Where's this app deployed?

**Reply by abhinavgurung** [reply]
in a Kubernetes cluster. AKS.

**Reply by icyJoseph** [reply]
I wonder if you can load other resources? Like images and such that might be in the public directory? Favicons etc?

**Reply by abhinavgurung** [reply]
No I cannot load other resources, not even favicons.

**Comment by pkhadka56** [comment]
`_next ` folder is created inside `.next/server/app/` directory if css or js files is missing. I am also getting the same issue. Any help?
